tasks/__init__.py

```diff
@@ -85,6 +85,11 @@ def adminserver(port=8001):
     cmd = '{} python manage.py runserver {} --nothreading'.format(env, port)
     run(cmd, echo=True, pty=True)
 
+@task
+def admin_task(args):
```

Comment:
Not sure why this is here. There is a tasks file in admin. Run `invoke admin.manage <command>` in order to run the admin manage task.